go.mod

```diff
@@ -3,8 +3,8 @@ module antrea.io/antrea
 go 1.19
 
 require (
-	antrea.io/libOpenflow v0.10.2
-	antrea.io/ofnet v0.7.2
+	antrea.io/libOpenflow v0.10.3
+	antrea.io/ofnet v0.7.3
```

Comment:
I'd like to double check with you about back-port, we need to bump up ofnet only for v1.11, right?

Reply:
No, we can back port it to both v1.11 and v1.12. As it is only bump up libOpenflow/ofnet versions, antrea v1.11 does not consume the packetIn2 APIs (it is consumed in v1.12) . There are also exisitng unmarshal issues in packetIn message when parsing IP/IGMP protocols in the previous versions which are not exposed yet, it is better to have this fix in the two versions to avoid unexpected failure in future.

Reply:
For 1.11, I guess you may need to create ofnet v0.6.11 and libOpenflow 0.9.3 to pick up bugfixes, as I remember there is incomptaible change in ofnet 0.7.0.